spec/cobrato/resources/remittance_cnab_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe Cobrato::Resources::RemittanceCnab do
+  let(:http)          { Cobrato::Http.new('6d60aa0fe1706fb3695ef0ad67f1da01') }
+  let(:entity_klass)  { Cobrato::Entities::RemittanceCnab }
+  let(:params) do
+    {
+      'charge_account_id' => '126',
+      'charge_ids'        => [1011, 1328]
+    }
+  end
+
+  subject { described_class.new(http) }
+
+  it 'has a instance of Cobrato::Http' do
+    expect(subject.http).to eq http
+  end
+
+  describe '#list' do
+    it 'returns an array of remittance_cnabs' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.